Jenkinsfile

```diff
@@ -1,213 +1,252 @@
                       }
                  }
              }
         }
         
         stage ('build') {
             steps {
-                withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin']) { 
+                withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin']) { 
                 sh """
                 #build opa-psm2
-		#ofi-install-path = /home/build/ofi-Install/libfabric
+                buildno=${env.BUILD_NUMBER}
                 rm -rf ${env.ofi_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER}
-	        mkdir -p ${env.ofi_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER}
+            mkdir -p ${env.ofi_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER}
                 ./autogen.sh
-		./configure --prefix=${env.ofi_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER} --with-psm2-src="$WORKSPACE/opa-psm2-lib"
+        ./configure --prefix=${env.ofi_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER} --with-psm2-src="$WORKSPACE/opa-psm2-lib"
                 make clean 
                 make && make install
                 echo "Hello World"
-	              """
+                  """
                 }
             }
         }
         stage('build-fabtests') {
             steps {
-                withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin']) { 
+                withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin']) { 
                 sh """
                     echo "to-do tests here"    
-                    rm -rf  ${env.fabtests_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER}
-            	      cd $WORKSPACE/fabtests
+                    #rm -rf  ${env.ofi_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER}/
+                      cd $WORKSPACE/fabtests
                     ./autogen.sh
-                    ./configure --prefix="${env.fabtests_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER}" --with-libfabric=${env.ofi_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER}
+                    ./configure --prefix="${env.ofi_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER}/" --with-libfabric=${env.ofi_install_dir}/${env.BRANCH_NAME}/${env.BUILD_NUMBER}
                     make clean
                     make && make install
                 """
                 }
             }
         }
-	    
+        
         stage ('build-shmem') {
             steps {
-              withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin']) {
+              withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin']) {
                 sh """
                  BranchName="${env.BRANCH_NAME}"
                  BuildNo="${env.BUILD_NUMBER}"
-                  chmod 777 contrib/Intel/JenkinsBuildScripts/Build-SHMEM.sh
-                 ./contrib/Intel/JenkinsBuildScripts/Build-SHMEM.sh \$BranchName \$BuildNo
+                  chmod 777 contrib/Intel/JenkinsBuildScripts/build_shmem.sh
+                 ./contrib/Intel/JenkinsBuildScripts/build_shmem.sh \$BranchName \$BuildNo
```

Comment:
If you do `sh <script>` or `bash <script>` you may be able to skip the chmod.

Reply:
okay, though the whole block is sh''', will give it a try.

Reply:
I think you need to prefix sh to every script that you want to run (if it doesn't execute permissions).